src/OSPSuite.InstallationValidator.Core/Domain/StartableProcess.cs

```diff
@@ -46,14 +46,15 @@ public void Dispose()
 
       public virtual void Wait(CancellationToken token)
```

Comment:
Is there a case where you would could start and not call wait?
Because it feels like wait is private and the token could be passe to Start(token); but I could be wrong